test/UserService.Mappers.UnitTests/ResponsesMappers/UserResponseMapperTests.cs

```diff
@@ -101,7 +103,7 @@ public void ShouldReturnUserModelWhenMappingValidDbUser()
             Assert.AreEqual(FirstName, resultUserModel.FirstName);
             Assert.AreEqual(LastName, resultUserModel.LastName);
             Assert.IsNull(resultUserModel.MiddleName);
-            Assert.AreEqual(Status, resultUserModel.Status);
+            Assert.AreEqual(status, resultUserModel.Status);
```

Comment:
all fields in this class start with a capital letter, and, I guess, you probably need to make this field with a capital letter

Reply:
done